src/Cafe/CafeSystem.cpp

```diff
@@ -68,6 +68,7 @@
 
 std::string _pathToExecutable;
 std::string _pathToBaseExecutable;
+std::string applicationName;
```

Comment:
This should not be a global. It's used only in `GetForegroundTitleName()` and it should be defined there.